perf/benchmark/setup_test.sh

```diff
@@ -54,14 +54,14 @@ function run_test() {
       --set server.injectL="${LINKERD_INJECT}" \
       --set domain="${DNS_DOMAIN}" \
       --set interceptionMode="${INTERCEPTION_MODE}" \
-          . > "${TMPDIR}"twopods.yaml
-  echo "Wrote file ${TMPDIR}twopods.yaml"
+          . > "${TMPDIR}""${NAMESPACE}".yaml
+  echo "Wrote file ${TMPDIR}${NAMESPACE}.yaml"
 
   # remove stdio rules
-  kubectl apply -n "${NAMESPACE}" -f "${TMPDIR}"twopods.yaml
-  kubectl rollout status deployment fortioclient -n twopods --timeout=1m
-  kubectl rollout status deployment fortioserver -n twopods --timeout=1m
-  echo "${TMPDIR}"twopods.yaml
+  kubectl apply -n "${NAMESPACE}" -f "${TMPDIR}""${NAMESPACE}".yaml
+  kubectl rollout status deployment fortioclient -n "${NAMESPACE}" --timeout=1m
+  kubectl rollout status deployment fortioserver -n "${NAMESPACE}" --timeout=1m
```

Comment:
we may only need to change these two lines? for the filename we can keep it is, or at least have "twopods" as prefix to make it more clear.

Also it should be "${TMPDIR}/twopods.yaml", missing a "/" in beween for existing code.

Reply:
The "${TMPDIR}" has a trailing "/" already.
The twopods.yml being renamed to "twopods-linkerd.yml" or "twopods-istio.yml", not the whole pathtwopods.yml, since they have "/" separator.

Reply:
from the logs it doesn't:
+ echo 'Wrote file /home/prow/go/src/istio.io/tools/perf/benchmark/tmptwopods.yaml'
Wrote file /home/prow/go/src/istio.io/tools/perf/benchmark/tmptwopods.yaml

so the path is mess up